tes-adapter/src/main/java/com/epam/pipeline/tesadapter/configuration/TesSwaggerConfig.java

```diff
@@ -26,10 +26,8 @@ public Docket api() {
 
     private ApiInfo getApiInfo() {
         return new ApiInfoBuilder()
-                .title("TES adapter API Documentation")
-                .description("Simple TES adapter application. " +
-                        "Without Security Configuration and Context")
-                .version("1.0.0")
+                .title("Task Execution Service (TES) API")
+                .description("Describes the REST API provided by a Cloud-Pipeline Task Execution Service")
```

Comment:
let's provide more specific information like:
`Represents implementation of GA4GH TES (https://github.com/ga4gh/task-execution-schemas) API for Cloud Pipeline`

Reply:
fixed